src/test/transaction_tests.cpp

```diff
@@ -94,6 +100,65 @@ std::string FormatScriptFlags(unsigned int flags)
     return ret.substr(0, ret.size() - 1);
 }
 
+bool CheckTxScripts(const CTransaction& tx, const std::map<COutPoint, CScript>& mapprevOutScriptPubKeys, const std::map<COutPoint, int64_t>& mapprevOutValues, const unsigned int& flags, const PrecomputedTransactionData& txdata, const std::string& strTest, const bool expect_valid)
```

Comment:
Please split this line (and perhaps comment what this function is doing)

Reply:
@glozow I find https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rf-in useful for when to pass "in" parameters by value versus by reference to const

Reply:
> Have line length guidelines ever been considered for this codebase? innocent

For the C++ part, the longer the better? (After many years of 80 or 100 char line limits, this surprised me at first, but for the C++ code I now actually prefer the lines to be unbroken more often than not.)

For the Python code, ISTM the unofficial practice is not PEP8 (79 chars) but more like 120 or 160 chars.

Reply:
I eyeballed ~100 chars, hopefully that's ok